src/app/src/test/kotlin/org/imperial/mrc/hint/integration/CleanDatabaseTests.kt

```diff
@@ -0,0 +1,48 @@
+package org.imperial.mrc.hint.integration
+
+import org.jooq.Table
+import org.imperial.mrc.hint.db.DbProfileServiceUserRepository
+import org.imperial.mrc.hint.db.JooqContext
+import org.imperial.mrc.hint.db.Tables
+import org.imperial.mrc.hint.helpers.tmpUploadDirectory
+import org.imperial.mrc.hint.unit.db.UserRepositoryTests
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import java.io.File
+import javax.sql.DataSource
+
+@ActiveProfiles(profiles = ["dev"])
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ExtendWith(SpringExtension::class)
```

Comment:
What does this `SpringExtension` do?

Reply:
Gives you access Autowiring etc, but I guess you don't need it if you're a SpringBootTest anyway..